src/Nova/NovaSentMail.php

```diff
@@ -57,7 +57,7 @@ public function fields(Request $request)
                     return trim(strip_tags($content));
                 })
                 ->alwaysShow(),
-            DateTime::make('Sent At', 'created_at')->format('M/d/Y h:mm:ss a'),
+            DateTime::make('Sent At', 'created_at')->format('M/D/Y h:mm:ss a'),
```

Comment:
@wilkerwma I think this change would mean we'd end up printing `Jan/Sat/2021`, I'm guessing we don't want that. Is this change related to a GH issue?

Reply:
@adammparker agreed, the format string is incorrect.
But no worries about this PR anymore. You can close it if you like.